src/utils/constants.js

```diff
@@ -0,0 +1,31 @@
+export const LOGO_URL = "https://d1tgh8fmlzexmh.cloudfront.net/ccbp-responsive-website/food-munch-img.png";
+
+export const CDN_URL = "https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_508,h_320,c_fill/"
+
+export const DEV_URL = "https://www.linkedin.com/in/rohith-goli-694b51170/"
+export const SOURCE_1_URL = "https://namastedev.com/namaste-react"
+export const SOURCE_2_URL = "https://in.linkedin.com/school/nxtwavetech/"
+
+export const SOCIAL_FB_URL = "https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_48,h_48/icon-facebook_tfqsuc"
+export const SOCIAL_PIN_INTEREST_URL = "https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_48,h_48/icon-pinterest_kmz2wd"
+export const SOCIAL_INSTA_URL = "https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_48,h_48/icon-instagram_b7nubh"
+export const SOCIAL_TWITTER_URL = "https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_48,h_48/icon-twitter_gtq8dv"
+
+export const SOCIAL_URL_LIST = [
+    {
+        socialUrl: SOCIAL_FB_URL,
+        id: "social-fb-url"
+    },
+    {
+        socialUrl: SOCIAL_PIN_INTEREST_URL,
+        id: "social-pin-interest-url"
+    },
+    {
+        socialUrl: SOCIAL_INSTA_URL,
+        id: "social-insta-url"
+    },
+    {
+        socialUrl: SOCIAL_TWITTER_URL,
+        id: "social-twitter-url"
+    }
+];
```

Comment:
exports are two types: default exports and named exports
As we need to export multiple values from this file, we are using named exports which will be imported  using
`import {---} from './-----`

In a file, only, one default export can exists which will be imported using
`import --- from ./----`
but several named exports can exists